src/main/java/uk/gov/digital/ho/hocs/casework/domain/repository/SuspensionRepository.java

```diff
@@ -0,0 +1,18 @@
+package uk.gov.digital.ho.hocs.casework.domain.repository;
+
+import org.springframework.data.repository.CrudRepository;
+import uk.gov.digital.ho.hocs.casework.domain.model.ActionDataSuspension;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+public interface SuspensionRepository extends CrudRepository<ActionDataSuspension, UUID> {
+
+    Optional<List<ActionDataSuspension>> findAllByCaseDataUuid(UUID caseUUID);
+
+    Optional<ActionDataSuspension> findByUuidAndCaseDataUuid(UUID uuid, UUID caseUUID);
+
+    Optional<List<ActionDataSuspension>> findAllByCaseDataUuidAndCaseTypeActionUuidAndDateSuspensionRemovedIsNull(UUID caseUUID, UUID caseActionTypeUUID);
```

Comment:
Only usage of this is in a method I'm not sure is needed so same applies to this

Reply:
I'll need to leave this in regardless as to whether the method that wraps it remains or not as there will need to be a check so as not to apply two suspensions to the deadline either way.